projects/demo/src/modules/components/message/index.html

```diff
@@ -0,0 +1,37 @@
+<tui-doc-page
+    header="Message"
+    package="KIT"
+    type="components"
+>
+    <ng-template pageTab>
+        <p>Message component is used to display message block.</p>
+
+        <tui-doc-example
+            *ngFor="let example of examples; let index = index"
+            [component]="index + 1 | tuiComponent"
+            [content]="index + 1 | tuiExample"
+            [heading]="example"
+            [id]="example | tuiKebab"
+        />
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <span
+                tuiMessage
```

Comment:
Do we need icons on message as well, maybe? Ask design team, I believe I saw in Figma that sometimes message has a chevron on the right